### PR TITLE
Simple fixes

### LINKED ISSUE #52: Treat CONTENT attribute not only from String elements (but others as well)
Currently, only CONTENT attributes from `String` are evaluated and realized in the resulting TEI. But ALTO has some other elements which may carry this attribute, most notably `HYP`.

### PR DESCRIPTION
**Pull request by bertsky**
Fixes #52 #51 #46

### CODE REVIEW

**Comment by bertsky** [comment]
Note: I have tested these, but not added new tests.

**Comment by bertsky** [comment]
(Removed the GH actions badge again – sry for the noise.)

**Comment by bertsky** [comment]
CI: I don't know how we can test this outside of master. But since it depends on the recent Makefile changes, I added it here.

**Review by kba — COMMENTED** [review]
This is superseeded by #58, right?

**Comment by bertsky** [comment]
> This is superseeded by #58, right?

Yes, it says it right there: `Expands on #56, additionally fixes #57.`

The idea was for @wrznr to be able to merge the smaller changeset first (but he fell ill and I had to keep going).

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,19 +1,54 @@
 # Python CircleCI 2.1 configuration file
 # for mets-mods2tei
 #
-# Check https://circleci.com/docs/2.1/language-python/ for more details
+# Check https://circleci.com/docs/2.0/language-python/ for more details
 #
 version: 2.1
```
Comment (wrznr) [inline .circleci/config.yml:6]: Maybe just `2`?

Comment (wrznr) [reply .circleci/config.yml:6]: Cf. https://circleci.com/docs/2.0/language-python/

**Review by wrznr — APPROVED** [review]

**Comment by wrznr** [comment]
Shall we still merge?

File: .circleci/config.yml (hunk 1 above)
Comment (bertsky) [reply .circleci/config.yml:6]: Both `2.1` and `2` yield a 404, hence the change.

Comment (bertsky) [reply .circleci/config.yml:6]: Oh sry, you were not referring to the comment, but to the `version` setting, let's see...

Comment (bertsky) [reply .circleci/config.yml:6]: No, since we rely on the codecov **orb**, we [require](https://circleci.com/docs/2.0/configuration-reference/#version) 2.1

**Comment by bertsky** [comment]
> Shall we still merge?

It will make #58 a little smaller, but still too large to view at once I'm afraid.

As you please.

**Pull request merged**